libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/navigation/OfflineRouteTest.kt

```diff
@@ -3,17 +3,28 @@ package com.mapbox.services.android.navigation.v5.navigation
 import android.content.Context
 import com.mapbox.api.directions.v5.DirectionsCriteria
 import com.mapbox.geojson.Point
-import com.mapbox.services.android.navigation.v5.utils.LocaleUtils
+import com.mapbox.services.android.navigation.v5.utils.extensions.inferDeviceLocale
 import io.mockk.every
 import io.mockk.mockk
+import io.mockk.mockkStatic
 import java.io.UnsupportedEncodingException
 import java.net.URLDecoder
 import java.util.Locale
 import org.junit.Assert.assertTrue
+import org.junit.BeforeClass
 import org.junit.Test
 
 class OfflineRouteTest {
 
+    companion object {
+
+        @BeforeClass
```

Comment:
As I understand, `BeforeClass` annotation makes function running before each test (like setup process)... Is  `mockkStatic("com.mapbox.services.android.navigation.v5.utils.extensions.ContextEx")` relates for all tests in this class, or only for the last test?

Reply:
`BeforeClass` only runs once in the context of `OfflineRouteTest` i.e. not before each test included in the test class (that'd be `@BeforeEach`). That being said, I think that if it's only needed for the last test we should added under the test to avoid confusion.

Reply:
cannot agree, this need for each test in this context. Lets keep `@BeforeClass`.

Reply:
If it's needed for every test, then `@BeforeClass` is 👌